packages/eslint-config-eslint/default.yml

```
@@ -157,6 +157,7 @@ rules:
     no-confusing-arrow: "error"
     no-console: "error"
     no-constructor-return: "error"
+    no-constant-binary-expression: "error"
```

```suggestion
    no-constant-binary-expression: "error"
    no-constructor-return: "error"
```

If we want to sort rules alphabetically, `no-constant-binary-expression` would be before `no-constructor-return`.